src/status_im/contexts/wallet/create_account/new_keypair/keypair_name/view.cljs

```diff
@@ -1,45 +1,67 @@
-            :char-limit      keypair-name-max-length
-            :auto-focus      true
-            :on-change-text  #(reset! keypair-name %)}]]]))))
+(def error-messages
+  {:length       (i18n/label :t/key-name-error-length)
+   :emoji        (i18n/label :t/key-name-error-emoji)
+   :special-char (i18n/label :t/key-name-error-special-char)})
+
+(defn view-internal
+  [{:keys [theme]}]
+  (let [[keypair-name set-keypair-name] (rn/use-state "")
+        customization-color             (rf/sub [:profile/customization-color])
+        [error set-error]               (rn/use-state false)]
+    [rn/view {:style {:flex 1}}
+     [floating-button-page/view
+      {:header [quo/page-nav
+                {:icon-name           :i/arrow-left
+                 :on-press            #(rf/dispatch [:navigate-back])
+                 :accessibility-label :top-bar}]
+       :footer [quo/bottom-actions
+                {:actions          :one-action
+                 :button-one-label (i18n/label :t/continue)
+                 :button-one-props {:disabled?           (or (pos? error)
+                                                             (<= (count keypair-name)
+                                                                 keypair-name-min-length))
+                                    :customization-color customization-color
+                                    :on-press            #(rf/dispatch [:wallet/new-keypair-continue
+                                                                        {:keypair-name
+                                                                         keypair-name}])}
+                 :container-style  style/bottom-action}]}
+      [quo/text-combinations
+       {:container-style style/header-container
+        :title           (i18n/label :t/keypair-name)
+        :description     (i18n/label :t/keypair-name-description)}]
+      [quo/input
+       {:container-style {:margin-horizontal 20}
+        :placeholder     (i18n/label :t/keypair-name-input-placeholder)
+        :label           (i18n/label :t/keypair-name)
+        :char-limit      keypair-name-max-length
+        :auto-focus      true
+        :on-change-text  (fn [value]
+                           (set-keypair-name value)
+                           (cond
+                             (> (count value) keypair-name-max-length)  (set-error :length)
+                             (validators/has-emojis? value)             (set-error :emoji)
+                             (validators/has-special-characters? value) (set-error :special-char)
+                             :else                                      (set-error nil)))
+        :error           error}]
+      (when error
+        [rn/view
+         {:style style/error-container}
+         [quo/icon :i/info {:color (colors/resolve-color :danger theme)}]
+         [quo/text
+          {:style (style/error theme)}
+          (get error-messages error)]])]]))
```

Comment:
If I'm not wrong, we have a component for this.

Reply:
Thanks @smohamedjavid, fixed it!